pluginconfig/commit.go

```diff
@@ -146,6 +146,12 @@ type CommitOffchainConfig struct {
 
 	// ChainFeeAsyncObserverSyncTimeout defines the timeout for a single sync operation (e.g. fetch token prices).
 	ChainFeeAsyncObserverSyncTimeout time.Duration `json:"chainFeeAsyncObserverSyncTimeout"`
+
+	// MaxRootsPerReport is the maximum number of roots to include in a single report.
+	MaxMerkleRootsPerReport uint64 `json:"maxRootsPerReport"`
+
+	// MultipleReports is a flag to enable/disable multiple reports per round.
+	MultipleReports bool `json:"multipleReports"`
```

Comment:
Following up from feedback from other around the config docs, lets add some more info on when these would be used. Also tiny rename to make boolean vars more obvious.

```suggestion
	// MaxRootsPerReport is the maximum number of roots to include in a single report.
	// Set this to 1 for destination chains that cannot process more than one commit root per report (e.g, Solana)
	// When set to 0, there is no maximum. <-- Is this the case?
	MaxMerkleRootsPerReport uint64 `json:"maxRootsPerReport"`

	// MultipleReports is a flag to enable/disable multiple reports per round.
	// This is typically set to true on chains that have `MaxMerkleRootsPerReport` equal to 1,
	// in order to remedy throughput issues of the commit plugin.
	// NOTE: this can only be set if RMNEnabled == false at the moment.
	MultipleReportsEnabled bool `json:"multipleReportsEnabled"`
```

Reply:
Good call, updated.